### PR TITLE
Prose copy edits

### PR DESCRIPTION
**Pull request by jeromefv**
## PR Checklist

- [x] Has pytest style unit tests (and `pytest` passes).
- [x] Is [Flake 8](https://flake8.pycqa.org/en/latest/) compliant (run `flake8` on changed files to check).
- [x] New features are documented, with examples if plot related.
- [x] Documentation is sphinx and numpydoc compliant (the docs should [build](https://matplotlib.org/devel/documenting_mpl.html#building-the-docs) without error).
- [x] Conforms to Matplotlib style conventions (install `flake8-docstrings` and `pydocstyle<4` and run `flake8 --docstring-convention=all`).
- [x] New features have an entry in `doc/users/next_whats_new/` (follow instructions in README.rst there).
- [x] API changes documented in `doc/api/next_api_changes/` (follow instructions in README.rst there).

### CODE REVIEW

File: tutorials/introductory/usage.py [hunk 1]
```diff
@@ -714,7 +714,7 @@ def my_plotter(ax, data1, data2, param_dict):
 #
 #   plt.plot(x, y, markevery=10)
 #
-# The markevery argument allows for naive subsampling, or an
+# The ``markevery`` argument allows for naive subsampling, or an
```
Comment (jklymak) [inline tutorials/introductory/usage.py:717]: I think we do kwargs as `*markevery*`.

**Review by jklymak — COMMENTED** [review]
I agree w/ all of these.  However, please remove the rst file from your commit (do `git rm doc/tutorials/introductory/usage.rst`).

File: tutorials/introductory/usage.py [hunk 2]
```diff
@@ -175,7 +175,7 @@
 plt.legend()
 
 ###############################################################################
-# Actually there is a third approach, for the case where you are embedding
+# In addition, there is a third approach, for the case when embedding
 # Matplotlib in a GUI application, which completely drops pyplot, even for
 # figure creation.  We won't discuss it here; see the corresponding section in
 # the gallery for more info (:ref:`user_interfaces`).
```
Comment (brunobeltran) [inline tutorials/introductory/usage.py:181]:
```suggestion
# In addition, there is a third approach, useful for embedding Matplotlib in a
# GUI application, which completely drops pyplot, even for figure creation.  We
# wont discuss it here; see the corresponding section in the gallery for more
# info (:ref:`user_interfaces`).
```

File: tutorials/introductory/usage.py [hunk 3]
```diff
@@ -357,16 +357,16 @@ def my_plotter(ax, data1, data2, param_dict):
 #
 # By default, Matplotlib should automatically select a default backend which
 # allows both interactive work and plotting from scripts, with output to the
-# screen and/or to a file, so at least initially you will not need to worry
+# screen and/or to a file, so at least initially, you will not need to worry
 # about the backend.  The most common exception is if your Python distribution
```
Comment (brunobeltran) [inline tutorials/introductory/usage.py:361]: The interjection feels like it's "at least initially".

As "fixed", the comma makes the sentence read like "so at least initially" is the interjection, but you can't remove that whole clause from the sentence and it still make sense. 

I would leave it out, personally, but you could also do ", so, at least initially," (eww) or, my prefered:

```suggestion
# screen and/or to a file, so you will not need to worry about the backend, at
# least initially. The most common exception is if your Python distribution
```

**Review by brunobeltran — CHANGES_REQUESTED** [review]
Couple of minor tweaks, plus the `->* fix from Jody, then this should just go in.

File: tutorials/introductory/usage.py (hunk 2 above)
Comment (QuLogic) [reply tutorials/introductory/usage.py:181]: s/wont/won't/

File: tutorials/introductory/usage.py (hunk 1 above)
Comment (jeromefv) [reply tutorials/introductory/usage.py:717]: I followed the formatting that was done on Line 708 and Line 712 for `markevery` in the paragraph. I can change all three instances to be formatted as *kwargs* as needed.

**Comment by jeromefv** [comment]
Thank you @jklymak and @brunobeltran for the suggestions! I've completed the requests and have moved forward with the process.

**Review by dstansby — APPROVED** [review]

**Review by jklymak — APPROVED** [review]

**Review by timhoffm — APPROVED** [review]

**Pull request merged**